Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1811,6 +1778,80 @@ void CGContextEOFillPath(CGContextRef context) {
 }
 #pragma endregion
 
+#pragma region Drawing Operations - CGImage
+
+/**
+ @Status Interoperable
+*/
+void CGContextDrawImage(CGContextRef context, CGRect rect, CGImageRef image) {
+    NOISY_RETURN_IF_NULL(context);
+    RETURN_IF(!image);
```

Comment:
any standard rule?

---
In reply to: [88942529](https://github.com/Microsoft/WinObjC/pull/1414#discussion_r88942529) [](ancestors = 88942529,88759177,88747402)